kratos/auth.yml

```diff
@@ -0,0 +1,64 @@
+version: '3.7'
+
+services:
+  postgres-kratos:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_USER=kratos
+      - POSTGRES_PASSWORD=secret
+      - POSTGRES_DB=kratos
```

Comment:
Add a volume for data.

Reply:
will add closer to production don't want to save users quite yet created issue for later, https://github.com/openfoodfacts/openfoodfacts-server/issues/6907